integration/querier_test.go

```diff
@@ -865,7 +864,6 @@ func TestQueryLimitsWithBlocksStorageRunningInMicroServices(t *testing.T) {
 				"-blocks-storage.bucket-store.sync-interval":           "1s",
```

Comment:
`t.Run(fmt.Sprintf("streaming enabled: %v", streamingEnabled), func(t *testing.T) {` above should probably be changed to be something like "store-gateway streaming enabled"

(also applies to test cases above)

Reply:
Done